src/utils/email/MailController.ts

```diff
@@ -0,0 +1,26 @@
+import { Request, Response } from 'express'
+import EmailService from './email.service'
+import { messages } from '../../db/messages'; 
+
+class MailController {
+    private emailService:EmailService;
+    constructor(){
+        this.emailService = new EmailService();
+    }
+
+    async sendEmail(req:Request,res:Response){
+        try{
+            const mail = this.emailService.send(req.body);
+            return messages.successMessage('success','sentMail',mail)
```

Comment:
Need to add `res.send(..)` to send the response to the client. If you don't need to send anything to the client, don't return a value.

```suggestion
            return res.send(messages.successMessage('success','sentMail',mail))
```